### ISSUE TITLE
Curator crashes with Python 3.9

### ISSUE THREAD

**Issue by sapensadler** [elastic/curator#1587]
When running curator with python 3.9 it crashes immediately with the following exception
```
Traceback (most recent call last):
  File "/usr/local/bin/curator", line 5, in <module>
    from curator.cli import cli
  File "/usr/local/lib/python3.9/site-packages/curator/__init__.py", line 7, in <module>
    from curator.utils import *
  File "/usr/local/lib/python3.9/site-packages/curator/utils.py", line 11, in <module>
    import elasticsearch
  File "/usr/local/lib/python3.9/site-packages/elasticsearch/__init__.py", line 24, in <module>
    from .client import Elasticsearch
  File "/usr/local/lib/python3.9/site-packages/elasticsearch/client/__init__.py", line 4, in <module>
    from ..transport import Transport
  File "/usr/local/lib/python3.9/site-packages/elasticsearch/transport.py", line 5, in <module>
    from .connection import Urllib3HttpConnection
  File "/usr/local/lib/python3.9/site-packages/elasticsearch/connection/__init__.py", line 2, in <module>
    from .http_requests import RequestsHttpConnection
  File "/usr/local/lib/python3.9/site-packages/elasticsearch/connection/http_requests.py", line 3, in <module>
    from base64 import decodestring
ImportError: cannot import name 'decodestring' from 'base64' (/usr/local/lib/python3.9/base64.py)
```
Downgrading to 3.8 fixes the issue. It appears you are using a deprecate function decodestring in the base64 library as per https://bugs.python.org/issue39351#:~:text=decodestring()%20are%20aliases%20deprecated,level%3A%20%3E%3E%3E%20base64.
### Expected Behavior
Curator should not crash

### Actual Behavior
It crashes.

### Steps to Reproduce the Problem
  1. Run `curator` using python 3.9

### Specifications

  - Version: 5.8.3
  - Platform: Ubuntu Docker

**Comment by untergeek** [comment]
You will note in [`setup.py`](https://github.com/elastic/curator/blob/V5.8.3/setup.py#L136) that 3.9 is not listed as supported yet. It will be added and tested at some point.

**Comment by untergeek** [comment]
I have to support multiple versions/releases of Python, and a jump to 3.9 practically forces the entire codebase to be changed or updated in ways that may prevent Curator from running on older python versions (including the EOL but still frequently used 2.7). I'm not sure I want to make that decision for all users of Curator.

**Comment by sapensadler** [comment]
Thanks for responding!

decodestring has been deprecated since Python 3.1.  Previous versions of curator will still work with python 2. I'd prefer support for newer versions than old EOL versions.

**Comment by untergeek** [comment]
> Previous versions of curator will still work with python 2. I'd prefer support for newer versions than old EOL versions.

Oh, as would I. When I make the cut to the next major release, I will cut off 2.x permanently. Until then, however, I'm going to keep supporting 2.7 (and only 2.7 from the 2.x branch). You would be surprised what people keep using "just because their distro provides it." I will create more work and headache for myself by blocking 2.7. I'll see if the most recent releases of 2.7 support `decodebytes`, and if that's the _only_ breaking thing stopping 3.9 adoption.

**Comment by sethmlarson** [comment]
It looks like this issue would be resolved by upgrading the `elasticsearch` version used within `install_requires`. Running this myself locally seems like curator would work on Python 3.9 without changes to curator code.

**Comment by kiddom-kq** [comment]
I can't comment on windows, but on mac it's trivial to have python 3.8 and 3.9 installed via `brew`. This should also be the case for *nix. 

This is all I needed to do:

```shell
# First, make sure you have python 3.8 installed
❯ brew search python3
==> Formulae
boost-python3                                     python@3.7                                        python@3.8 ✔                                      python@3.9 ✔

If you meant "python3" specifically:
It was migrated from homebrew/cask to homebrew/core.

# Then use the the full path to the 3.8 binary to run the venv module
❯ /usr/local/Cellar/python@3.8/3.8.7_2/bin/python3 -m venv ./venv
❯ source ./venv/bin/activate
# Then check to confirm your version
❯ python3
Python 3.8.7 (default, Feb  3 2021, 06:31:03) 
[Clang 12.0.0 (clang-1200.0.32.29)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
❯ pip3 install elasticsearch-curator
<...>
❯ curator
Usage: curator [OPTIONS] ACTION_FILE

Error: Missing argument "action_file".
```

**Comment by untergeek** [comment]
I will release an updated version of Curator which uses the newest `elasticsearch-py` client as soon as I get some spare cycles.

**Comment by kiddom-kq** [comment]
@untergeek No worries / rush!  I posted the work around precisely because we (consumers of OSS) have to work around the schedules / energy / priorities / life ...etc of the core devs/maintainers.

The work around (basically use Py3.8 or older) should be pretty assessable to anybody that's also able to run the latest python3.9 releases.

**Comment by untergeek** [comment]
This has been corrected as of 5.8.4, and 3.9 is in the test suite for Travis CI
